### ISSUE TITLE
Bluebird Promise library complains about unhandled errors

### ISSUE THREAD

**Issue by artzstudio** [artzstudio/meanify#4]
User experiencing console error logs due to not handling promise error generated by Mongoose.  

Suggested workaround: Automattic/mongoose#3931

**Comment by neagle** [comment]
The workaround seems fine. I haven't checked yet whether I can implement an easy workaround on my side to suppress the warning. It seems _slightly_ irritating that Mongoose returns a promise no matter what, but I can understand that not being at the top of Mongoose's priorities, since the behavior of other libraries is to not care about unhandled rejections.

**Comment by artz** [comment]
Indeed, indeed.  I'll let you know once I've implemented the stubs.
